tests/ociswrapper/wrapper/handlers/handler.go

```diff
@@ -201,3 +202,54 @@ func CommandHandler(res http.ResponseWriter, req *http.Request) {
 	exitCode, output := ocis.RunCommand(command, stdIn)
 	sendCmdResponse(res, exitCode, output)
 }
+
+func OcisServiceHandler(res http.ResponseWriter, req *http.Request) {
+	serviceName := req.PathValue("service")
+	envMap := []string{fmt.Sprintf("OCIS_EXCLUDE_RUN_SERVICES=%s", serviceName)}
+
+	if req.Method == http.MethodPost {
+		// restart oCIS without service that need to start separately
+		success, _ := ocis.Restart(envMap)
+		if success {
+			var envBody map[string]interface{}
+			var serviceEnvMap []string
+
+			if req.Body != nil && req.ContentLength > 0 {
+				var err error
+				envBody, err = parseJsonBody(req.Body)
+				if err != nil {
+					sendResponse(res, http.StatusBadRequest, "Invalid json body")
+					return
+				}
+			}
+
+			for key, value := range envBody {
+				serviceEnvMap = append(serviceEnvMap, fmt.Sprintf("%s=%v", key, value))
+			}
+
+			log.Println(fmt.Sprintf("[ociswrapper] Starting oCIS service %s......", serviceName))
```

Comment:
Please, use the local logger: https://github.com/owncloud/ocis/blob/d7c47719c960ba781be0422335e09556ef937521/tests/ociswrapper/log/log.go